mkdocs.yml

```diff
@@ -7,36 +7,39 @@ repo_url: https://github.com/k8sgpt-ai/k8sgpt
 edit_uri: https://github.com/k8sgpt-ai/docs/edit/main/docs/
 
 nav:
-  - Getting Started: 
-      - Overview: index.md
-      - Getting Started Guide: getting-started/getting-started.md
-      - Installation: getting-started/installation.md
-      - In-Cluster Operator: getting-started/in-cluster-operator.md
-  - Tutorials:
-      - Overview: tutorials/index.md
-      - Content Collection: tutorials/content-collection/content-collection.md
-  - Reference:
-      - CLI:
-          - Overview: reference/cli/index.md
-          - Integration and Filter: reference/cli/filters.md
-      - Operator:
-          - Overview: reference/operator/overview.md
-      - Guidelines & Community:
-          - Guidelines: reference/guidelines/guidelines.md
-          - Community: reference/guidelines/community.md
-          - Privacy: reference/guidelines/privacy.md
-  - Explanation: explanation/index.md
+    - Getting Started:
+          - Overview: index.md
+          - Getting Started Guide: getting-started/getting-started.md
+          - Installation: getting-started/installation.md
+          - In-Cluster Operator: getting-started/in-cluster-operator.md
+    - Tutorials:
+          - Overview: tutorials/index.md
+          - Content Collection: tutorials/content-collection/content-collection.md
+    - Reference:
+          - CLI:
+                - Overview: reference/cli/index.md
+                - Integration and Filter: reference/cli/filters.md
+          - Operator:
+                - Overview: reference/operator/overview.md
+          - Guidelines & Community:
+                - Guidelines: reference/guidelines/guidelines.md
+                - Community: reference/guidelines/community.md
+                - Privacy: reference/guidelines/privacy.md
+    - Explanation: explanation/index.md
 
 theme:
-  name: material
-  language: "en"
-  logo: imgs/icon-logo-01.svg
-  features:
-    - navigation.tabs
-    - navigation.tabs.sticky
-    - navigation.sections
-    - content.tabs.link
+    name: material
```

Comment:
Why did you change the mkdocs file? Did you test it is still working correctly?

Reply:
Apologies for the confusion, it looks like VSCode somehow added an extra double space for each line. The only intended change here is the addition of 

```yaml
markdown_extensions:
  - admonition
 ```
More information [Click Here](https://squidfunk.github.io/mkdocs-material/reference/admonitions/).

Here is a screenshot of the docs working with the Info block.

![image](https://github.com/k8sgpt-ai/docs/assets/7347011/30f588e5-8e1f-4fba-b47e-5d44bbe4755f)


I'll update the file accordingly.